domains/qefw.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "imdakiki",
+           "email": "skullkid73@outlook.com",
+           "discord": "898255050592366642"
+        },
+    
+        "record": {
+            "CNAME": "kikisdevelopment.github.io"
```

Comment:
Page seems to give a 404 error, if you have the site hosted then please send a link to it